hummedia/api.py

```diff
@@ -35,6 +35,16 @@ def Collection(collection,id=None):
 	else:
 		return endpoint_404()
 
+@app.route('/video/expired', methods=['GET', 'OPTIONS'])
+@crossdomain(origin=CROSS_DOMAIN_HOSTS,headers=['Origin','x-requested-with','accept'],credentials=True)
+def getExpiredMedia():
```

Comment:
I think we should restrict it to admins only. I'm not sure how to do this using the framework.